plugins/flytekit-whylogs/setup.py

```diff
@@ -4,11 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = [
-    "protobuf>=3.15,<4.0.0"
-    "whylogs",
-    "whylogs[viz]"
-]
+plugin_requires = ["protobuf>=3.15,<4.0.0" "whylogs", "whylogs[viz]"]
```

Comment:
you need a comma between the first two items.

Reply:
thanks for the catch!